.prettierrc.json

```diff
@@ -1,6 +1 @@
-{
-  "semi": false,
-  "singleQuote": true,
-  "proseWrap": "always",
-  "trailingComma": "all"
-}
+"@netlify/eslint-config-node/.prettierrc.json"
```

Comment:
Re-use shared Prettier configuration.

We want to use a `.prettierrc.json` and `.eslintrc.js` as opposed to using a `package.json` properties, because I personally found some tools (like VSCode) to not recognize Prettier/ESLint configuration when inside a `package.json` sometimes.